android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/Result.java

```diff
@@ -125,19 +125,7 @@ public void writeToParcel(Parcel dest, int flags) {
         dest.writeValue(app);
         dest.writeValue(wup);
 
-        dest.writeBooleanArray(new boolean[]{
-                ready_to_report,
-                got_server_ack,
-                suspended_via_gui,
-                project_suspended_via_gui,
-                coproc_missing,
-                gpu_mem_wait,
-                active_task,
-                supports_graphics,
-                too_large,
-                needs_shmem,
-                edf_scheduled
-        });
+        dest.writeBooleanArray(new boolean[]{ready_to_report, got_server_ack, suspended_via_gui, project_suspended_via_gui, coproc_missing, gpu_mem_wait, active_task, supports_graphics, too_large, needs_shmem, edf_scheduled});
```

Comment:
This is a type of change that makes things **clearly** worse. The human editor had clear and justified intentions (with the line breaks) which should be preserved.